src/api/app/views/webui/package/_files_view.html.erb

```diff
@@ -9,7 +9,12 @@
       </tr>
       </thead>
       <tbody>
-        <%= render partial: 'file', collection: @files, cached: true, locals: { project: @project, package: @package, expand: @expand } %>
+        <%- file_locals = {
+          project: @project, package: @package, expand: @expand, is_current_rev: @is_current_rev,
+          can_modify: User.current.can_modify?(@package), nobody: User.current.is_nobody?
+        } %>
+        <%= render partial: 'file', collection: @files, cached: proc { |file| [file, file_locals].hash }, locals: file_locals
+        %>
```

Comment:
about time the bento pages for packages and projects find their rest :(

Reply:
What do you mean?

Reply:
the whole theory behind "let's keep bento and bootstrap alive in parallel for a while" was based on the assumption that every developer can convert one action a day. As this assumption failed, my recommendation would be to give up and make project and package pages bootstrap only (risking a mixed environment for everyone) - and continue with beta flag only for pages under sprint development.

But that's unrelated to this PR - it just makes every fix in views a little more complicated